autonomy/replay/utils.py

```diff
@@ -21,12 +21,14 @@
 
 import json
 from pathlib import Path
+from typing import Any, Dict
 
 from autonomy.deploy.constants import PERSISTENT_DATA_DIR, TM_STATE_DIR
 
 
 def fix_address_books(build_dir: Path) -> None:
     """Update address books in data dump to use them in replays."""
+    # TODO: add json validation and error handling
```

Comment:
This is a big issue throughout. Various config files have no validation and then also no error handling when loading/accessing